### ISSUE TITLE
Desktop app cut off

### ISSUE THREAD

**Issue by 2stay** [klinker-apps/messenger-issues#709]
**Device with issue:** <Windows 10  Desktop>

Prompted for update install on desktop PC. Installed Pulse SMS 3.0.0. Pulse SMS did not re-launch after install of update. Manually relaunched, and Pulse SMS was signed out. Refreshed through Android phone app, and signed in again on desktop. All custom theme settings did not sync, and vertical dots in upper right corner "Account, Settings, Logout, etc." are not sizing with window -- cut off in corner, and mouse over barely allows clicking. 

**Additional information:**
![capture](https://user-images.githubusercontent.com/41457703/42979413-dc229e02-8b86-11e8-9a3f-3b335e75d59c.PNG)

 screenshot attached.

**Comment by klinker24** [comment]
The web app backing the desktop app was updated to pulsesms.app, which is why you needed to sign in again. Your settings were lost because of the account refresh when you forgot the password. There is an issue here for storing more of those settings and restoring them, already.

Is the last option just a window size thing? That is the actionable item here that can be fixed. So, if you increase the window size, it works correctly?

**Comment by 2stay** [comment]
Hi Luke,

Thanks for the quick reply. I hadn't thought resetting the password would mean losing theme settings, as I recall at one time the desktop app syncing to the settings of the phone app? I just thought it would sync again after signing back in, but it didn't. In any case, you are correct, as the desktop update forced me to sign in again, and I needed to reset the password to do that because I couldn't remember it.

With regard to the window sizing, the only way to see the vertical dots completely is to maximize the window (full screen). Otherwise, no matter how you drag horizontally or diagonally to the size (of say a Skype messenger window) desired, it doesn't show completely.

Love the app though, which is why I had bought the lifetime subscription. Keep up the great work! Cheers.


Get Outlook for Android

________________________________

**Comment by klinker24** [comment]
Is this still an issue for you, on the latest version of the desktop app? I have not been able to reproduce it, but Electron has gotten numerous under the hood updates that probably affected this.

**Comment by 2stay** [comment]
Yes.

Get Outlook for <https://aka.ms/ghei36> Android<https://aka.ms/ghei36>

________________________________

**Comment by klinker24** [comment]
Yes it is fixed, or yes it is still an issue?

**Comment by 2stay** [comment]
Still an issue.


Get Outlook for <https://aka.ms/ghei36> Android<https://aka.ms/ghei36>

________________________________

**Comment by joselevelsup** [comment]
I'm getting a similar issue...or maybe not. But for me, the desktop only shows like half of the app. I toggled the dev tools and it seems like its not part of the app. 
<img width="1680" alt="screen shot 2018-12-01 at 9 44 18 am" src="https://user-images.githubusercontent.com/10066172/49329544-886e8a00-f54e-11e8-963b-e5c9ec8e301f.png">

**Comment by klinker24** [comment]
I am sure you can just close the app and re-open it. You can close it by right clicking the icon on the mac launcher, and selecting quit. The red x button won't close the app.

**Comment by joselevelsup** [comment]
I've already tried that AND even Cmd+q. The restarting but nothing.

**Comment by klinker24** [comment]
I don't know, sorry. I haven't had any other reports of this nor have I seen it myself. Reinstall, I suppose.

**Comment by 2stay** [comment]
Desktop version is the latest in the repository. I keep updating, but the bug isn't fixed. I can still click on the vertical ellipses, but only by moving my mouse to the farthest your right corner until the pointer changes. If I make expand to full desktop, I can see it. This is only in minimized mode for use in a similar sized window to Skype, so you can be productive in other windows and still use Pulse.

**Comment by joselevelsup** [comment]
Yeah. I tried reinstalling but nothing. Still the same cutoff.

**Comment by Vi1i** [comment]
I can help debug on my side if needed, in reference to #815.

**Comment by jsseidel** [comment]
I came to report that for a while the message bubble at the bottom is cut off running the latest Pulse under HiDPI (don't know if it's an issue in non-HiDPI). This is what I see at the bottom of the application window. Tried resizing, etc. Always cut off. Running under Gnome 3. Stock ubuntu 18.04.

![Screenshot from 2019-04-03 16-38-57](https://user-images.githubusercontent.com/11260601/55511747-528dff00-562f-11e9-921e-09687c8c48c6.png)
